src/model_reformulation_nosnoc.m

```diff
@@ -1037,7 +1037,7 @@
     if simple_v0_guess
         v0 = zeros(n_x,1);
     else
-        [v0,~] = (f_x_fun(x0,z0_all,u0,[p_global_val; p_time_var_val(:,1)]));
+        [v0,~] = (f_x_fun(x0,z0_all,u0,[p_global_val; p_time_var_val(:,1)], v0_global));
```

Comment:
hmm, the names of v0 and v0_global are a bit confusing, maybe adding a comment would be instructive.
ones are stage "velocity values", others are global time independent variables

Reply:
Yea a comment for now but I still don't love the `v_global` name as a whole.

Reply:
I am more than open to better suggestions 😁 
We were discussing for a while how to name it and we wanted to be consistent with the global user parameters.
But somewhat we run out of letters.